llpc/test/shaderdb/PipelineVsFs_RelocConst.pipe

```diff
@@ -0,0 +1,105 @@
+
+// This test case checks that descriptor offset relocation works for buffer descriptors in a vs/fs pipeline.
+; BEGIN_SHADERTEST
+; RUN: amdllpc -spvgen-dir=%spvgendir% -use-relocatable-shader-elf -o test_pipeline_out_relocconst.elf %gfxip %s && llvm-objdump -triple=amdgcn -mcpu=gfx900 -d test_pipeline_out_relocconst.elf | FileCheck -check-prefix=SHADERTEST %s && rm test_pipeline_out_relocconst.elf
+; SHADERTEST-LABEL: 0000000000000000 _amdgpu_vs_main:
+; SHADERTEST: s_mov_b32 s[[RELOREG:[0-9]+]], 12 //{{.*}}
+; SHADERTEST: s_load_dwordx4 s[{{.*}}:{{.*}}], s[{{.*}}:{{.*}}], s[[RELOREG]] //{{.*}}
+; SHADERTEST: {{[0-9A-Za-z]+}} _amdgpu_ps_main:
+; END_SHADERTEST
```

Comment:
Could you please augment the test to have relocations in both the VS and the PS, to make sure the relocation offsets are lined up correctly?

Reply:
Done.

Reply:
Done.